code/ignacio/python/compute-ari_lab.py

```diff
@@ -0,0 +1,46 @@
+with open('wizard-of-oz.txt', encoding="utf8" ) as f:
+    lines = f.readlines()
+    
+book = ''
+for item in lines:
+    book += item
+book_count = len(book)
+
+sentences = []
```

Comment:
This should technically be defined as an int since that is what sentences will become, but it's ok.